Crash when typing in empty tag

I accidentally deleted everything from the tag. (Foolish me, I wish undo worked in tags..)

I typed and things were sad.

```
Text.Select: DoubleClick returned 0, 59
panic: runtime error: index out of range

goroutine 68 [running]:
main.(*Window).Commit(0xc000497400, 0xc000497418)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/wind.go:580 +0x2c1
main.(*Window).SetTag1(0xc000497400)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/wind.go:493 +0x5a
main.(*Window).SetTag(0xc000497400)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/wind.go:476 +0x8c
main.cut(0xc000497418, 0xc000497418, 0x0, 0x101, 0x0, 0x0)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/exec.go:323 +0x5a0
main.(*Text).Type(0xc000497418, 0xc000000044)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/text.go:922 +0x230
main.(*Window).Type(0xc000497400, 0xc000497418, 0x44)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/wind.go:431 +0x37
main.(*Row).Type(0x1373c00, 0xc000000044, 0x529, 0x16c, 0xc000497401)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/row.go:288 +0x133
main.keyboardthread(0xc0000c20c0)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/acme.go:468 +0x1c3
created by main.main
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/acme.go:173 +0x921
```

--- a/rune.go
+++ b/rune.go
@@ -31,27 +31,6 @@ func runeEq(r, s []rune) bool {
 	return true
 }
 
-func runesplitN(buf []rune, sep []rune, nl int) [][]rune {
-	linestart := 0
-	lines := [][]rune{}
-	for i, r := range buf {
-		for _, se := range sep {
-			if r == se {
-				line := buf[linestart:i]
-				lines = append(lines, line)
-				linestart = i + 1
-			}
-			if len(lines) >= nl {
-				break
-			}
-		}
-	}
-	if linestart != len(buf) {
-		lines = append(lines, buf[linestart:]) // trailing chunk
-	}
-	return lines
-}
-
 func isIn(r []rune, s rune) bool {
 	for _, c := range r {
 		if s == c {

--- a/wind.go
+++ b/wind.go
@@ -574,7 +574,10 @@ func (w *Window) Commit(t *Text) {
 	}
 	r := make([]rune, w.tag.Nc())
 	w.tag.file.b.Read(0, r)
-	filename := string(runesplitN(r, []rune(" \t"), 1)[0])
+	filename := string(r)
+	if i := strings.IndexAny(filename, " \t"); i >= 0 {
+		filename = filename[:i]
+	}
 	if filename != w.body.file.name {
 		seq++
 		w.body.file.Mark()

